src/initFirebase.tsx

```diff
@@ -0,0 +1,7 @@
+import firebase from 'firebase';
+
+import {firebaseConfig } from './constants/firebaseConfig';
```

Comment:
The same thing with a whitespace, you should either delete it from the right or add it to the left.

Reply:
fixed it. Thanks